src/templates/_pages/page_layout.twig

```diff
@@ -0,0 +1,125 @@
+{# page_layout.twig #}
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+    <meta name="description" content="A demo app"/>
+    <meta name="author" content="Dan Dediu"/>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    {% block head %}
+        <link rel="stylesheet" href="css/styles.css"/>
+        <title>{% block title %}{% endblock %}</title>
+    {% endblock %}
```

Comment:
please take the link and title away from `block head`
they are default and should be applied everywhere
the task was to support any eventually new links
for that, please add a new block below this line - `{% block additional_styles %}{% endblock %}`